src/selectors/markdownSelectors.test.js

```diff
@@ -3,15 +3,19 @@ import { getMarkdown } from './markdownSelectors';
 describe('markdown selectors tests', () => {
   it('gets markdown text from state', () => {
     const state = {
-      markdown: 'some string'
+      markdownReducer: {
```

Comment:
A reducer should be a pure function, not something that's referenced in state.

Reply:
🤠

Reply:
c'mon we're just learning this stuff!! (trying to be as explicit as possible without confusing ourselves with naming conventions like markdown.markdown)